src/polycubectl/cliargs/cliargs.go

```diff
@@ -304,22 +305,22 @@ func ParseCLIArgs(args []string) (*CLIArgs, error) {
 }
 
 func (cli *CLIArgs) buildURL() string {
-	var url string
+	var url_str string
 
 	for _, str := range cli.PathArgs {
-		url += str + "/"
+		url_str += url.PathEscape(str) + "/"
```

Comment:
Add a comment why the above changes are needed. This applies also to line 316.

Reply:
@mauriciovasquezbernal